### ISSUE TITLE
Refactor request: Add para doc elements

### ISSUE THREAD

**Issue by wmjordan** [JosefPihrt/Roslynator#350]
The requested refactor wraps new-lines-separated lines with `para` elements.

Before:
````C#
/// <summary>
/// The comment contains multiple lines.
///
/// Next line
/// is broken into two lines.
///
/// More line.
/// </summary>
````

After:
````C#
/// <summary>
/// <para>The comment contains multiple lines.</para>
/// <para>Next line
/// is broken into two lines.</para>
/// <para>More line.</para>
/// </summary>
````

**Comment by JosefPihrt** [comment]
My understanding is that you are asking for an analyzer, am I right?

**Comment by wmjordan** [comment]
I am asking for a refactor.
When we call for the QuickFix within the `summary` element, it provides such a refactor to change new lines into `para` elements.

**Comment by JosefPihrt** [comment]
But you want to fix all occurrences similar to your code sample?

If so then it should be an analyzer (with a code fix that would add `para` elements, of course).

**Comment by wmjordan** [comment]
Well, you are right. An analyzer helps too.
So we can fix similar occurrences at one time.

**Comment by JosefPihrt** [comment]
fixes with c783ef3c264820166ed6a3ab8b8c79bc11683ca7
